CHANGES.md

```diff
@@ -80,6 +81,184 @@
 * [Version 1.0.1](#version-101)
 * [Version 1.0](#version-10)
 
+## Version 3.4.0 (September 5, 2025)
+
+OpenEXR v3.4 introduces support for *lossless compression with High
+Throughput JPEG-2000 (HTJ2K)* encoding:
+
+* The new HTJ2K compressor uses the *High-Throughput (HT) block coder*, and
+  it supports the full range of OpenEXR features, including 16-bit and
+  32-bit floating-point image channels, both scanline and tiled.
+
+* The HT block coder is standardized in [Rec. ITU-T
+  T.814](https://loc.gov/preservation/digital/formats/fdd/fdd000566.shtml)
+  and [ISO/IEC 15444-15](https://www.iso.org/standard/76621.html). It
+  is *royalty-free*, widely used in cinema and distribution servicing,
+  and implemented in both commercial and open-source toolkits.
+
+* Integration with OpenEXR uses the
+  [OpenJPH](https://github.com/aous72/OpenJPH) open-source
+  library. For ease in managing the dependency, the OpenEXR CMake
+  configuration supports *automatically fetching and building `OpenJPH`
+  internally*, or linking against an *external installation*.
+
+* OpenEXR supports two new compression types: 
+
+  - `htj2k32` -- encodes/decodes chunks of 32 scanlines, offering
+    significantly faster encoding and decoding (4-6x in some cases).
+
+  - `htj2k256` -- encodes/decodes chunks of 256 scanlines, producing
+    slightly smaller file sizes.
+
+* :warning: This is a *backwards-compatible extension* to the OpenEXR
+  file format. Files written with OpenEXR v3.4 will be readable by
+  applications built against previous releases, _unless_ they use the
+  new `htj2k32` or `htj2k256` compression options.
```

Comment:
We could also say that in general software compiled with OpenEXR-3.4 will be able to read HTJ2K compressed OpenEXRs without any code changes. Software that writes files may automatically support the new type, but may  need a small update to make the new type available as a user option.